web/controllers/panel_survey_controller.ex

```diff
@@ -1,48 +1,67 @@
 defmodule Ask.PanelSurveyController do
   use Ask.Web, :api_controller
 
-  alias Ask.{Repo, Survey}
+  alias Ask.{PanelSurvey, Repo}
 
   def index(conn, %{"project_id" => project_id}) do
-    latest_panel_surveys =
-      load_project(conn, project_id)
-      |> assoc(:surveys)
-      |> where([s], s.latest_panel_survey)
-      |> Repo.all()
+    project = conn
+    |> load_project(project_id)
 
-    panel_surveys = Enum.map(latest_panel_surveys, fn s -> prepare_panel_survey(s) end)
+    panel_surveys = project
+      |> assoc(:panel_surveys)
+      |> Repo.all()
 
     render(conn, "index.json", panel_surveys: panel_surveys)
   end
 
+  def create(conn, %{"project_id" => project_id, "panel_survey" => panel_survey_params}) do
```

Comment:
So, currently this is not exposed right? because in the router the `new` and `edit` are excluded.. Or am I missing something?

Reply:
I think this is just following the Surveda routes standard.

The available routes for panel surveys and their corresponding controller functions are:

```
GET /api/v1/projects/:project_id/panel_surveys Ask.PanelSurveyController :index
GET /api/v1/projects/:project_id/panel_surveys/:id Ask.PanelSurveyController :show
POST /api/v1/projects/:project_id/panel_surveys Ask.PanelSurveyController :create
PATCH /api/v1/projects/:project_id/panel_surveys/:id Ask.PanelSurveyController :update
PUT /api/v1/projects/:project_id/panel_surveys/:id Ask.PanelSurveyController :update
DELETE /api/v1/projects/:project_id/panel_surveys/:id Ask.PanelSurveyController :delete
```

Reply:
Answering your question: yes, the `create` function of this controller is exposed on this route: `POST /api/v1/projects/:project_id/panel_surveys`